api/managementpb/external.proto

```diff
@@ -9,16 +9,25 @@ import "google/api/annotations.proto";
 import "inventorypb/agents.proto";
 import "inventorypb/services.proto";
 import "managementpb/metrics.proto";
+import "managementpb/service.proto";
 
 // Add External
 
 message AddExternalRequest {
-  // Node identifier on which a external exporter is been running. Required.
-  string runs_on_node_id = 1 [
-    (validator.field) = {
-      string_not_empty: true
-    }
-  ];
+  // Node identifier on which an external exporter is been running.
+  // runs_on_node_id always should be passed with node_id.
+  // Exactly one of these parameters should be present: node_id, node_name, add_node.
+  string runs_on_node_id = 1;
+  // Node name on which a service and node is been running.
```

Comment:
Very confusing sentence...
Did you mean `external exporter` instead of `service and node`?
@BupycHuk, could you please clarify/fix?